vic_run/src/runoff.c

```diff
@@ -39,48 +39,51 @@ runoff(cell_data_struct  *cell,
        soil_con_struct   *soil_con,
        double             ppt,
        double            *frost_fract,
-       int                dt,
+       double             dt,
```

Comment:
@tbohn : Could you review runoff.c to make sure it does what it needs to. It had a hardwired hourly time loop before